ledger-core/v2/vanilla/iokit/teezero.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
should be `teeskip.go`/`tee.go`, not teezero?

Reply:
renamed